google/api_core/exceptions.py

```diff
@@ -441,6 +450,10 @@ def from_grpc_status(status_code, message, **kwargs):
         GoogleAPICallError: An instance of the appropriate subclass of
             :class:`GoogleAPICallError`.
     """
+
+    if isinstance(status_code, int):
+        status_code = _INT_TO_GRPC_CODE[status_code]
```

Comment:
`_INT_TO_GRPC_CODE` only exists `if grpc`. Is this a problem? Also, do we want to get a with a default value (perhaps just return status code again?) in case the value isn't present?

Reply:
Accepted @tseaver's suggestions (`_INT_TO_GRPC_CODE` always exists and  default value is `status_code`)